android/gradle/.android/advancedFeatures.ini

```diff
@@ -0,0 +1 @@
+GLESDynamicVersion = on
```

Comment:
How is that file used? I think this is for the emulator, right? how does my local installation find this?

Reply:
This file is needed for the emulator to enable OpenGL ES 3.1 I believe.
I found the solution here: https://stackoverflow.com/a/44552548/2794211

Reply:
But this file should be in `~/.android/`. That means users need to install this manually/make the changes manually. So we need to document this but not add this file.

Reply:
Oh that's right, I had read it wrong! My problem solved itself when I added the file, but if the folder it should live in is ~/.android, it is probably something else that solved my problem... I will try again to see if it works without the file and I guess we can remove it if it doesn't help. I'll see if there is another special configuration I made for the emulator in Android Studio and document it. I'll do that next week probably.

Reply:
I have removed the file as it was not used anyway.